src/cryptography/hazmat/primitives/twofactor/utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import unicode_literals
+
+import base64
+
+from six.moves.urllib.parse import quote, urlencode
+
+
+__all__ = ['get_provisioning_uri']
```

Comment:
I'm ok adding support for this. I still have some feedback on the substance
of the overall patch.

On Wed, May 27, 2015 at 12:55 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/cryptography/hazmat/primitives/twofactor/utils.py
> <https://github.com/pyca/cryptography/pull/1990#discussion_r31155244>:
>
> > @@ -0,0 +1,50 @@
> > +from __future__ import unicode_literals
> > +
> > +import base64
> > +
> > +from six.moves.urllib.parse import quote, urlencode
> > +
> > +
> > +__all__ = ['get_provisioning_uri']
>
> This isn't needed (and we probably don't want to encourage import *)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1990/files#r31155244>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084